index.js

```diff
@@ -3,9 +3,18 @@ var express = require('express');
 var app = express();
 var port = process.env.PORT || 8080;
 
+const config = require('./config.js');
```

Comment:
Where the App is hosted on Heroku, We can't put in a config file, but we can put in some .env variables. Would it be better to put these in as these instead of relying on this file to exist?

Reply:
Easiest way is probably just to put a config.js in the repo which refers to the process.env variables you want to use for staging in Heroku.